src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -60,24 +61,24 @@ std::string postEntities
 {
   Entity*  eP = &parseDataP->ent.res;
 
-
-  // 00. Temp debug
-  // LM_M(("Presenting"));
```

Comment:
A little off-topic, but... are is the LM_M/LM_F stuff "controlled"? Is there any other remaining LM_M/LM_F? I'm a bit nervous since I discovered a forgotten LM_M/LM_F was consuming 25% of contextBroker CPU self-time during recent performance tests...